docs/components/list.md

```diff
@@ -362,13 +362,13 @@ Token                                 | Default value
 </style>
 <md-list>
   <md-list-item headline="Wide Cat">
-    <img slot="start-image" src="https://placekitten.com/200/100">
+    <img slot="start" style="width: 40px" src="https://placekitten.com/200/100">
```

Comment:
Can you move this into the style block instead?

```
<style>
  :root {
    ...
  }

  img {
    width: 40px;
  }
</style>
```

Reply:
Did so.